### PR TITLE
TW-898 Various logic bugfixes

### PR DESCRIPTION
**Pull request by keshan3262**

### CODE REVIEW

**Review by svjatoslavtt — APPROVED** [review]

File: src/screens/earn/farm-item/farm-item.tsx [hunk 1]
```diff
@@ -135,11 +139,19 @@ export const FarmItem: FC<Props> = ({ farm, lastStakeRecord }) => {
...
+              <FormattedAmount isDollarValue amount={depositAmountAtomic} style={styles.attributeValue} />
+            )}
           </View>
           <View style={styles.flex}>
             <Text style={styles.attributeTitle}>Claimable rewards:</Text>
-            <FormattedAmount isDollarValue amount={claimableRewardsAtomic} style={styles.attributeValue} />
+            {stakeIsLoading ? (
+              <Text style={styles.attributeValue}>---</Text>
+            ) : (
+              <FormattedAmount isDollarValue amount={claimableRewardsAtomic} style={styles.attributeValue} />
```
Comment (herkoss) [inline src/screens/earn/farm-item/farm-item.tsx:153]: Should we move this logic to separate component to avoid code duplicating?

**Review by herkoss — APPROVED** [review]
GJ!

**Review by herkoss — APPROVED** [review]
GJ!

**Review by svjatoslavtt — APPROVED** [review]

**Review by IrynaKhyzhynska — APPROVED** [review]
android done 1.15_788

**Review by vladvn1905 — APPROVED** [review]
ios done 1.15_788

**Review by svjatoslavtt — APPROVED** [review]

**Review by herkoss — APPROVED** [review]

**Review by herkoss — APPROVED** [review]

**Review by svjatoslavtt — APPROVED** [review]

**Review by IrynaKhyzhynska — APPROVED** [review]
android done 1.15__802

**Review by vladvn1905 — APPROVED** [review]
ios done 1.15_802

**Pull request merged**